public/content/roadmap/fusaka/peerdas/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: PeerDAS
+description: Learn about PeerDAS as part of the Fusaka Ethereum protocol upgrade
+lang: en
+---
+
+The Ethereum protocol is undergoing its most significant scaling upgrade since the [introduction of blob transactions with EIP-4844](/roadmap/danksharding/). As part of the [Fusaka upgrade](/roadmap/fusaka/), PeerDAS introduces a new way of handling blob data, delivering roughly an order-of-magnitude increase in **[data availability (DA)](/developers/docs/data-availability/) ** capacity for L2s.
```

Comment:
```suggestion
The Ethereum protocol is undergoing its most significant scaling upgrade since the [introduction of blob transactions with EIP-4844](/roadmap/danksharding/). As part of the [Fusaka upgrade](/roadmap/fusaka/), PeerDAS introduces a new way of handling blob data, delivering roughly an order-of-magnitude increase in **[data availability (DA)](/developers/docs/data-availability/)** capacity for L2s.
```